deepchem/models/callbacks.py

```diff
@@ -90,4 +90,4 @@ def __call__(self, model, step):
     if model.wandb or (model.wandb_logger is not None):
```

Comment:
It not directly related to this PR, but shouldn't this line be just

```python
if model.wandb_logger is not None:
```

You reference `model.wandb_logger` inside the block, so you'll get an exception if it's `None`.